internal/fvalidate/verapdf_validator_test.go

```diff
@@ -94,3 +95,23 @@ func TestValidate(t *testing.T) {
 		})
 	}
 }
+
+func TestVeraPDFVersion(t *testing.T) {
+	t.Parallel()
+
+	v := fvalidate.NewVeraPDFValidator("echo", logr.Discard())
+	version, err := v.Version()
+
+	assert.NilError(t, err)
+	assert.Assert(t, strings.Contains(version, "echo (GNU coreutils)"))
+}
+
+func TestVeraPDFVersionError(t *testing.T) {
+	t.Parallel()
+
+	v := fvalidate.NewVeraPDFValidator("false", logr.Discard())
+	version, err := v.Version()
+
+	assert.Error(t, err, "exit status 1")
+	assert.Equal(t, version, "")
+}
```

Comment:
Just realized about these test. How hard is to mock the command execution? I'd prefer something along those lines that actually running these commands.

Reply:
I was pondering that as well... I'll look into mocking as that should be more consistent.

Reply:
I've added mocking of the command execution.